engine/components/sql/src/test/scala/pl/touk/nussknacker/sql/utils/StandaloneConfigCreator.scala

```diff
@@ -0,0 +1,56 @@
+package pl.touk.nussknacker.sql.utils
+
+import io.circe.Decoder
+import io.circe.generic.JsonCodec
+import io.circe.generic.semiauto._
+import net.ceedubs.ficus.Ficus._
+import net.ceedubs.ficus.readers.ArbitraryTypeReader._
+import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.process._
+import pl.touk.nussknacker.engine.standalone.api.StandaloneSinkWithParameters
+import pl.touk.nussknacker.engine.standalone.utils.JsonStandaloneSourceFactory
+import pl.touk.nussknacker.engine.util.process.EmptyProcessConfigCreator
+import pl.touk.nussknacker.sql.db.pool.DBPoolConfig
+import pl.touk.nussknacker.sql.service.DatabaseLookupEnricher
+
+class StandaloneConfigCreator extends EmptyProcessConfigCreator {
```

Comment:
let's add TODO to extract to some standalone tests (not-existing module yet)

Reply:
done